stan/math/prim/prob/von_mises_cdf.hpp

```diff
@@ -0,0 +1,124 @@
+#ifndef STAN_MATH_PRIM_PROB_VON_MISES_CDF_HPP
+#define STAN_MATH_PRIM_PROB_VON_MISES_CDF_HPP
+
+#include <stan/math/prim/fun/constants.hpp>
+#include <stan/math/prim/prob/normal_cdf.hpp>
+#include <stan/math/prim/fun/modified_bessel_first_kind.hpp>
+#include <cmath>
+
+namespace stan {
+namespace math {
+
+namespace internal {
+
+template <typename T_x, typename T_k, typename T_p>
+return_type_t<T_x, T_k, T_p> von_mises_cdf_series(const T_x& x, const T_k& k,
+                                                  const T_p& p) {
+  using stan::math::cos;
+  using stan::math::sin;
+
+  double pi = stan::math::pi();
+  auto s = sin(x);
+  auto c = cos(x);
+  auto sn = sin(p * x);
+  auto cn = cos(p * x);
+
+  return_type_t<T_x, T_k, T_p> R = 0;
+  return_type_t<T_x, T_k, T_p> V = 0;
```

Comment:
```suggestion
  using return_t = return_type_t<T_x, T_k, T_p>;
  return_t R(0.0);
  return_t V(0.0);
```

I think we usually prefer calling the constructor over the assignment operator